lib/ng-json2js.js

```diff
@@ -26,7 +37,7 @@ module.exports = (function () {
                     });
             file.path = file.path + '.js';
 
-            done(util.format(TEMPLATE, jsonPath, componentName, content));
+            done(util.format(TEMPLATE, componentName, jsonPath, componentName, content, componentName, componentName));
```

Comment:
This is getting unreadable with this many parameters... How about turning `TEMPLATE` into a function and then just invoke it here with correct parameters? the function should be invoked in the following way:
```js
template({
    jsonPath: jsonPath,
    componentName: componentName,
    content: content,
});
```